platforms/core-execution/build-cache-local/src/integTest/groovy/org/gradle/caching/local/internal/AbstractBuildCacheCleanupIntegrationTest.groovy

```diff
@@ -187,25 +164,18 @@ abstract class AbstractBuildCacheCleanupIntegrationTest extends AbstractIntegrat
         assertCacheWasCleanedUpSince(lastCleanupCheck)
 
         where:
-        buildCacheCleanup | deprecatedCacheCleanup | effectiveCleanup | scenario
-        null              | null                   | 7                | "default period when not explicitly configured"
-        2                 | null                   | 2                | "configured period for build cache cleanup"
-        null              | 2                      | 2                | "configured period for deprecated build cache cleanup"
-        1                 | 10                     | 10               | "configured period for deprecated build cache cleanup when both are configured"
+        buildCacheCleanup | effectiveCleanup | scenario
+        null              | 7                | "default period when not explicitly configured"
+        2                 | 2                | "configured period for build cache cleanup"
     }
 
     def "produces reasonable message when cache retention is too short (#days days)"() {
-        settingsFile << """
-            buildCache {
-                local {
-                    removeUnusedEntriesAfterDays = ${days}
-                }
-            }
-        """
+        initializeHome()
+        withBuildCacheRetentionInDays(days)
+
         expect:
         runAndFail()
-        // TODO: Change name to Build cache, since Directory build cache is not the only cache now
-        failure.assertHasCause("Directory build cache needs to retain entries for at least a day.")
+        failure.assertHasDescription("buildCache cannot be set to retain entries for $days days.")
```

Comment:
:x: This message looks weird. Shouldn't it say "Build cache `local` cannot be set to..." instead? It definitely shouldn't say `buildCache`.

Reply:
It's  default message from this "cleanup strategy". It's concatenated as:
`cacheName +  cannot be set to retain entries for...`

I can fix that to uppercase cache name

Reply:
Fixed